crates/matrix-sdk/src/lib.rs

```diff
@@ -48,6 +48,9 @@ pub mod sync;
 #[cfg(feature = "experimental-sliding-sync")]
 pub mod sliding_sync;
 
+#[cfg(feature = "widget-api")]
+pub mod widget_api;
+
 #[cfg(feature = "e2e-encryption")]
 pub mod encryption;
```

Comment:
Can you remove the blank lines between all of the module declarations, and instead have just one to separate the `mod`s from the `pub use`s?

Reply:
Sure. One question though: should I also remove the empty lines around the `sliding_sync`. Rationale: I added `pub mod widget_api` there since it was following the experimental sliding sync module which also was separated by spaces.

I.e. it was:

```rust
#[cfg(feature = "experimental-oidc")]
pub mod oidc;
pub mod room;
pub mod sync;

#[cfg(feature = "experimental-sliding-sync")]
pub mod sliding_sync;

#[cfg(feature = "e2e-encryption")]
pub mod encryption;
```

So I placed the `pub mod widget_api` right after the `pub mod sliding_sync`.